PerfTools/JeProf/interface/jeprof.h

```diff
@@ -0,0 +1,4 @@
+
+namespace jeprof {
+  void makeHeapDump();
```

Comment:
I think the function should take the file name as an argument.

Reply:
OK